api/index.md

```diff
@@ -9,105 +9,30 @@ has_children: true
 The [Data Commons Graph](https://datacommons.org) aggregates data from many
 different [data sources](https://datacommons.org/datasets) into a single
 database. Data Commons is based on the data model used by
-[schema.org](https://schema.org), for more information see [our guide to the data model](/data_model.html).
+[schema.org](https://schema.org); for more information, see [the guide to the data model](/data_model.html).
 
-The **Data Commons API** is a set of APIs that allow developers to
-programmatically access the data in Data Commons.
-Data Commons provides APIs in each of the different views
-(graph, timeline, etc.) for use in different contexts. More specifically, we provide
-1. **REST** APIs that can be used by javascript based applications
-1. **Pandas** APIs for use in Python notebooks
-1. **Google Sheets** APIs for use in Google Sheets
-1. **Python** APIs
-1. **SQL** (Coming soon)
+The **Data Commons API** allows developers to programmatically access the data in Data Commons.
```

Comment:
This is very well written. I just wonder if it's helpful to users to see the different endpoints highlighted (maybe at least bold/italics if we want to remove the bullet points)?

Reply:
Maybe even with links to the docsite pages for each endpoint

Reply:
I've changed the docs per this suggestion. Let me know what feels best to you!

Reply:
looks good!